pkg/nodediscovery/nodediscovery.go

```diff
@@ -582,8 +583,8 @@ func (n *NodeDiscovery) mutateNodeResource(nodeResource *ciliumv2.CiliumNode) er
 				nodeResource.Spec.ENI.VpcID = c.ENI.VpcID
 			}
 
-			if len(c.ENI.ExcludeInterfaceTags) > 0 {
-				nodeResource.Spec.ENI.ExcludeInterfaceTags = c.ENI.ExcludeInterfaceTags
```

Comment:
Why this removal? Looks like an accidental change

Reply:
Its is an accidental change from cherry picking changes between 1.10 and master. Sorry about this. Will push a fix shortly.